### ISSUE TITLE
Any

### ISSUE THREAD

**Issue by Ashrafty** [amirk3321/flutter_ad_manager_web#2]
Is there any hopes of updating this package i want to use it in my project...i just want to know if you are willing to support it long term

**Comment by thunderbug1** [comment]
I am also interesting in this repo, is it still maintained or are you using a different way to display ads now? If so, would you mind sharing it?

**Comment by amirk3321** [comment]
I am planning to update this repository, and I will go live with the new update as soon as possible. I am actively working on it. This repository is currently functioning to display AdSense ads in Flutter web. If you check the [Inkblot Art](https://inkblot.art) website, which We have developed entirely using Flutter technology, you will get an idea. I am doing my best to provide the community with the best ads package. 

Live Preview:
<img width="1439" alt="Screenshot 2023-11-16 at 4 32 20 PM" src="https://github.com/amirk3321/web-socket-example/assets/10207753/f99aae3c-4931-449c-98d4-b7ba5d782277">

**Comment by amirk3321** [comment]
> Is there any hopes of updating this package i want to use it in my project...i just want to know if you are willing to support it long term

Yes, I will provide long-term support for this package. Due to some busy contracts, I haven't updated it yet, but I plan to update it very soon.

**Comment by Ashrafty** [comment]
thank you so much...also do you have plans to support other ad networks
